processing/src/main/java/io/druid/query/Queries.java

```diff
@@ -33,6 +34,7 @@
 
 /**
  */
+@PublicApi
 public class Queries
```

Comment:
There's a judgement call here: extension implementors could always copy/paste the code from Queries rather than linking to it directly, and there is cost to the core project for having too many public apis. But I think that in this case it's worth it.